src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -29,7 +29,9 @@ struct JnienvInitializeArgs {
 		public int             localRefsAreIndirect;
 		public int             grefGcThreshold;
 		public IntPtr          grefIGCUserPeer;
-		public int             isRunningOnDesktop;
```

Comment:
I wonder if this could *possibly* break the Designer...  Just a rando fear.

Reply:
Designer tests *did* fail in this build, although it doesn't appear to be related (but might be?):

```
1) Failed : Xamarin.AndroidDesigner.Tests.Moyeu.CanLoadLayoutsInProject
  /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ADT-0/RT-Moyeu-1/Moyeu/Resources/layout/InfoPaneLayout.axml
  Expected: no item equal to "MockView"
  But was:  < "MockView" >
  First non-matching item at index [0]:  "MockView"
  at Xamarin.AndroidDesigner.Tests.RegressionTest.CanLoadLayoutsInProject () [0x00000] in <055988ff4d7c4b6f95c58bd23af8596c>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.InvokeMoveNext (System.Object stateMachine) [0x00000] in <208e3d8480c243b7804b22b6a5eb72fc>:0
```

Reply:
And they failed again in the same way after reverting this change, so this is not the cause